nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/document/AddPermissionTest.java

```diff
@@ -421,6 +426,37 @@ public void shouldAddPermissionForUserWhenBlockingInheritance() throws Operation
         assertExpectedPermissions(acl, params, "existingUser1");
     }
 
+    @Test
+    public void shouldAddPermissionForUsersWhenUsingChainExtension() throws OperationException {
+        // Mock the users params
+        List<String> users = Arrays.asList("jchastain", "rdeniro", "dwashington");
+        users.forEach(user -> when(userManager.getUserModel(user)).thenReturn(new SimpleDocumentModel(user)));
+
+        DocumentModel doc = session.getDocument(new PathRef("/src"));
+        OperationContext ctx = new OperationContext(session);
+        ctx.setInput(doc);
+
+        Object object = automationService.run(ctx, "testAddPermissionToUsers");
+        assertThat(object, instanceOf(DocumentModel.class));
+
+        DocumentModel documentModel = (DocumentModel) object;
+
+        ACL acl = documentModel.getACP().getACL("local");
+        assertNotNull("Local ACL cannot be null !", acl);
```

Comment:
Not strictly needed here, we know that the `"local"` ACL exists.

Reply:
yes, but i think with my test i am ensuring that the permission is correctly created with the good param that i give (in my case **local**).  what do you think :)

Reply:
Sure!

Reply:
I would remove the unnecessary text, `assertNotNull(acl)` will be quite explicit if it fails.

Reply:
yep,